org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationMatching.cs

```diff
@@ -0,0 +1,718 @@
+                {
+                    worksheet.Cells[row, 15].Value = hubspotVal.ToString("MM/dd/yyyy");
+                }
+            }
+            worksheet.Cells[row, 16].Value = person.ModifiedDateTime.Value.ToString("MM/dd/yyyy");
+
+            //Add Rock Id
+            worksheet.Cells[row, 17].Value = person.Id;
+
+
+            return worksheet;
+        }
+
+        private bool CustomEquals(string a, string b)
+        {
+            if ( !String.IsNullOrEmpty(a) && !String.IsNullOrEmpty(b) )
+            {
+                return a.ToLower() == b.ToLower();
+            }
+            return false;
+        }
+
+    }
+
+    [DebuggerDisplay("Label: {label}, FieldType: {fieldType}")]
+    public class HubspotProperty
+    {
+        public string name { get; set; }
+        public string label { get; set; }
+        public string fieldType { get; set; }
+        public string groupName { get; set; }
+    }
+
+    public class HubspotPropertyUpdate
+    {
+        public string property { get; set; }
+        public string value { get; set; }
+    }
+
+    public class HSContactProperties
+    {
+        public string createdate { get; set; }
+        public string email { get; set; }
+        public string firstname { get; set; }
+        public string lastname { get; set; }
+        public string lastmodifieddate { get; set; }
+        private string _phone { get; set; }
+        public string phone
+        {
+            get
+            {
+                return !String.IsNullOrEmpty(_phone) ? _phone.Replace(" ", "").Replace("(", "").Replace(")", "").Replace("-", "") : "";
+            }
+            set
+            {
+                _phone = value;
+            }
+        }
+        public string rock_person_id { get; set; }
+
+        public string rock_record_status { get; set; }
+
+        public string has_potential_rock_match { get; set; }
+
+        public override string ToString()
+        {
+            return "CreatedDate: " + createdate + "; LastModifiedDate: " + lastmodifieddate + "; First: " + firstname + "; Last:" + lastname + "; Email: " + email + "; Phone: " + phone + "; RockId: " + rock_person_id + "; RockStatus: " + rock_record_status + "; RockPotentialMatch: " + has_potential_rock_match;
+        }
+    }
+
+    [DebuggerDisplay("Id: {id}, Email: {properties.email}")]
+    public class HSContactResult
+    {
+        public string id { get; set; }
+        public HSContactProperties properties { get; set; }
+        public string archived { get; set; }
+
+        public override string ToString()
+        {
+            return "Id: " + id + "; Properties: " + properties.ToString();
+        }
+    }
+    public class HSResultPaging
+    {
+        public HSResultPagingNext next { get; set; }
+    }
+    public class HSResultPagingNext
+    {
+        public string link { get; set; }
+    }
+    public class HSContactQueryResult
+    {
+        public List<HSContactResult> results { get; set; }
```

Comment:
This is used in both jobs. I'd prefer that it either be duplicated in both or that it be in a separate file that both jobs can access. Public classes should not live in jobs.